src/cloud/docker/docker-containers.md

```diff
@@ -0,0 +1,131 @@
+
+## Container Volumes
+
+Docker volumes are used to maintain data throughout the lifecycle of the docker containers.  These volumes can be defined in several ways:
+
+- in docker-compose.yml or other docker-compose files
+- in the Dockerfile from the [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-docker)
+- The upstream docker image
+
+Most of these volumes are not interacted with, they are just used by the containers and will follow the lifecycle of docker-compose. The only exception to this is the magento-sync directory which is used by Mutagen or Docker-Sync to transport data into the containers from the host OS.
+
+You can remove all data, and "start fresh" by using:
+```
+ bin/docker down
+ # this triggers a docker-compose command which removes volumes
+ # docker-compose down -v
+```
+This will not remove the magento-sync volume. As mentioned above this is a special volume which requires creating/destroying on it's own.  
+
+Without this volume created you will see the following error message.
+```
+ERROR: Volume magento-sync declared as external, but could not be found. Please create the volume manually using `docker volume create --name=magento-sync` and try again.
+```
+
+
+
+## Container Logs
+
+All containers log using Dockers built in logging method. The easiest way to interface with this is to use the docker-compose command and view the logs.
+
+The following example will follow the tls container log files, allowing you to see all requests that are passing through, or errors it has had.
+```sh
+docker-composer logs -f tls
+```
+
+
+## Sendmail service
+
+You can send emails from your Docker environment when you enable `sendmail` in the `docker-compose.yml` configuration file:
+
+Sendmail can slow down the instancing of containers, it is best to not use it on the cli containers if possible.
+
+```yaml
+ENABLE_SENDMAIL=true
+```
+
+
+
+[Docker architecture]: {{ site.baseurl }}/common/images/cloud/docker-topology.png
+[Database container]: {{site.baseurl}}/cloud/docker/docker-database.html
+[CLI containers]: {{site.baseurl}}/cloud/docker/docker-cli.html
+[Web containers]: {{site.baseurl}}/cloud/docker/docker-php.html
+[Developer Mode]: {{site.baseurl}}/cloud/docker/docker-mode-developer.html
+[File Synchronization]: {{site.baseurl}}/cloud/docker/docker-syncing-data.html
+[docker-repo]: https://github.com/magento/magento-cloud-docker
+[elastic]: https://hub.docker.com/r/magento/magento-cloud-docker-elasticsearch
+[mariadb]: https://hub.docker.com/_/mariadb
+[nginx]: https://hub.docker.com/r/magento/magento-cloud-docker-nginx
+[node]: https://hub.docker.com/_/node/]
+[rabbitmq]: https://hub.docker.com/_/rabbitmq
+[php-image]: https://hub.docker.com/r/magento/magento-cloud-docker-php
+[redis]: https://hub.docker.com/r/magento/magento-cloud-docker-redis

```

Comment:
We use official Redis image

Reply:
Good catch, also took out that extra trailing ] in this.